src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json

```diff
@@ -1,11 +1,11 @@
 {
   "Color": "Red",
   "Region": "West US",
-  "FeatureManagement": {
+  "feature_management": {
```

Comment:
Why?

Reply:
This is automatically modified during the test running, not manually modified by me